docs/_docs/installation/installing-using-zip.adoc

```diff
@@ -0,0 +1,100 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+= Installing Using ZIP Archive
+
+== Prerequisites
+
+Ignite was tested on:
+
+include::../includes/prereqs.adoc[]
+
+
+== Installing Using ZIP Archive
+
+
+. Download the archive with the Ignite CLI tool:
++
+[tabs]
+--
+tab:Unix[]
+[source,shell]
+----
+curl -L "https://www.apache.org/dyn/mirrors/mirrors.cgi?action=download&filename=ignite/3.0.0-alpha5/apache-ignite-3.0.0-alpha5.zip" -o apache-ignite-3.0.0-alpha5.zip
+----
+
+tab:Windows (PowerShell)[]
+[source,shell]
+----
+include::../includes/ignite-cli-download-windows.adoc[tag=command,indent=0]
+----
+
+tab:Windows (CMD)[]
+[source,shell]
+----
+include::../includes/ignite-cli-download-windows.adoc[tag=command,indent=0]
+----
+--
+
+. Unpack the archive:
++
+[tabs]
+--
+tab:Unix[]
+[source,shell]
+----
+unzip apache-ignite-3.0.0-alpha5.zip && cd apache-ignite-3.0.0-alpha5
+----
+
+
+tab:Windows (PowerShell)[]
+[source,shell]
+----
+Expand-Archive apache-ignite-3.0.0-alpha5.zip -DestinationPath . ; cd apache-ignite-3.0.0-alpha5
+----
+
+tab:Windows (CMD)[]
+[source,shell]
+----
+tar -xf apache-ignite-3.0.0-alpha5.zip & cd apache-ignite-3.0.0-alpha5
+----
+--
+
+. Add your installation directory to the PATH environment variable:
++
+[tabs]
+--
+tab:Unix[]
+[source,shell]
+----
+echo 'export PATH="'`pwd`':$PATH"' >> ~/.bash_profile && source ~/.bash_profile  && source ignite_completion.sh
```

Comment:
I think zip flow has been changed since alpha5, you can look at some references in DEVNOTES